rrs/views.py

```diff
@@ -0,0 +1,1005 @@
+# rrs-web - views
+#
+# Copyright (C) 2014 Intel Corporation
+#
+# Licensed under the MIT license, see COPYING.MIT for details
+#
+# SPDX-License-Identifier: MIT
+
+import urllib
+
+import csv
+from django.http import HttpResponse
+
+from datetime import date, datetime
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+from django.views.generic import TemplateView, ListView, DetailView, RedirectView
+from django.core.urlresolvers import resolve, reverse, reverse_lazy
+from django.db import connection
+from django.contrib import messages
+
+from layerindex.models import Recipe, StaticBuildDep, Patch
+from rrs.models import Release, Milestone, Maintainer, RecipeMaintainerHistory, \
+        RecipeMaintainer, RecipeUpstreamHistory, RecipeUpstream, \
+        RecipeDistro, RecipeUpgrade, MaintenancePlan, RecipeSymbol
+
+
+
+class FrontPageRedirect(RedirectView):
+    permanent = False
+
+    def get_redirect_url(self):
+        maintplan = MaintenancePlan.objects.first()
+        if not maintplan:
+            raise Exception('No maintenance plans defined')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.



Similar issue at line numbers 38 and 41.